sdk/eventgrid/eventgrid/tests.yml

```diff
@@ -5,5 +5,3 @@ stages:
     parameters:
       PackageName: "@azure/eventgrid"
       ServiceDirectory: eventgrid
-      MatrixFilters:
-        - DependencyVersion=^$
```

Comment:
Just curious, what was this filter for and why don't we need it now? @ellismg

Reply:
I also do not know what the purpose of this was.  I think that it ended up creeping in when I started on the implementation way back, since I just took an existing project's files and did some minimal renaming.  As I was working on this, I noticed that it was present whereas many other projects didn't have it and so I figured I would just remove it and see what happens.  Everything continued to work and this makes things here be in-line with other projects, so I figured I just keep it removed.

Sorry for such a disappointing answer.

Reply:
@ellismg Spoke to Ben and found out that this is a filter used to not run min-ma tests. And to enable min-max tests you just need to removed that filter, which you did :)